src/amo/components/CollectionManager/index.js

```diff
@@ -346,19 +346,21 @@ export class CollectionManagerBase extends React.Component<Props, State> {
           />
         </div>
 
-        <ReactCSSTransitionGroup
+        <TransitionGroup
           className="NoticePlaceholder"
-          component="div"
-          transitionName="NoticePlaceholder-transition"
-          transitionEnterTimeout={MESSAGE_FADEOUT_TIME}
-          transitionLeaveTimeout={MESSAGE_FADEOUT_TIME}
         >
-          {this.state.addonAddedStatus === ADDON_ADDED_STATUS_SUCCESS && (
+          <CSSTransition
+            component={null}
+            timeout={MESSAGE_FADEOUT_TIME}
```

Comment:
I think we can remove the constant `MESSAGE_FADEOUT_TIME` and direct pass the value `450` to timeout

Reply:
no, we should use constants